docs/docsite/rst/porting_guide_2.4.rst

```diff
@@ -131,6 +131,28 @@ Developers:
 * Any callbacks inheriting from other callbacks might need to also be updated to contain the same documented options
   as the parent or the options won't be available.  This is noted in the developer guide.
 
+Template lookup plugin
```

Comment:
Can we give more or a clue in the heading, perhaps?
`Template lookup plugin: Escaping strings`